### PR TITLE
Feature/jjdd8 fpo 6

### PR DESCRIPTION
**Pull request by DariuszCzajkowski**

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/BookRepository.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.infoshareacademy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BookRepository {
+
+    public static List<Book> bookRepository = new ArrayList<>();
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/BookRepository.java]: to prywatne :)

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**